pom.xml

```diff
@@ -1110,7 +1122,11 @@ under the License.
 							<groupId>org.apache.maven.plugins</groupId>
 							<artifactId>maven-surefire-plugin</artifactId>
 							<configuration>
-								<excludedGroups>org.apache.flink.testutils.junit.FailsOnJava17</excludedGroups>
+								<excludedGroups>
+									${github-actions.surefire.excludedGroups},
+									org.apache.flink.testutils.junit.FailsOnJava11,
+									org.apache.flink.testutils.junit.FailsOnJava17,
```

Comment:
Wondering if this is correct; isn't it possible for a test to fail on Java 11 but work on Java 17? Specifically cases for JDK 17 exclusive features.

Reply:
Fair point. Right now, there's only a single test that's JDK-specific. That's why I didn't bother that much. But I added a commit now that removes this dependency between the different JDK version supports. I also fixed the scenario where we want to run the AdaptiveScheduler with a JDK other than 8.